### PR TITLE
Added intel project oneTBB

### PR DESCRIPTION
**Pull request by sudhirverma**
Fix: #374

### CODE REVIEW

**Review by asa1997 — CHANGES_REQUESTED** [review]
@sudhirverma Version file name is incorrect

File: bes_theme/assets/data/version_details/374-oneTBB-Versiondetails.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[
+    {
+        "version": "v2021.7.0",
+        "release_date": "",
```
Comment (asa1997) [inline bes_theme/assets/data/version_details/374-oneTBB-Versiondetails.json:4]: @sudhirverma Please provide the release date

**Review by asa1997 — CHANGES_REQUESTED** [review]
@sudhirverma Please address the comments

File: bes_theme/assets/data/version_details/374-oneTBB-Versiondetails.json (hunk 1 above)
Comment (sudhirverma) [reply bes_theme/assets/data/version_details/374-oneTBB-Versiondetails.json]: done

**Review by asa1997 — APPROVED** [review]
@sudhirverma LGTM

**Pull request merged**